### PR TITLE
[#3671] Update links in contributing docs

### PR DESCRIPTION
**Pull request by ljupcovangelski**

### CODE REVIEW

File: docs/docs/guides/contributing-components.md [hunk 1]
```diff
@@ -3,12 +3,6 @@ title: Contributing Components
 sidebar_label: Contributing Components
 ---
 
-:::warning
```
Comment (ljupcovangelski) [inline docs/docs/guides/contributing-components.md:6]: Why was this removed? The repo is still empty. I think we should put our own components first, so that they serve as an example.

**Review by ljupcovangelski — COMMENTED** [review]
I think we should keep the comment that this is still WIP.

File: docs/docs/guides/contributing-components.md (hunk 1 above)
Comment (chrismatix) [reply docs/docs/guides/contributing-components.md:6]: It read a bit confusing and imprecise: Of course, you can already contribute components. It's just that the catalog mechanism is not complete yet. I updated it to reflect that.

Comment (ljupcovangelski) [reply docs/docs/guides/contributing-components.md:6]: Ok, feel free to approve, as I cannot do it :smile:

**Review by chrismatix — APPROVED** [review]
